crates/re_types_builder/src/codegen/rust/serializer.rs

```diff
@@ -40,7 +40,7 @@ pub fn quote_arrow_serializer(
 
     let quoted_bitmap = |var| {
         quote! {
-            let #var: Option<::arrow2::bitmap::Bitmap> = {
+            let #var: Option<::re_types_core::external::arrow2::bitmap::Bitmap> = {
```

Comment:
Let's just `use ::re_types_core::external::arrow2;` at the top instead - this is way too hairy

Reply:
That's already the case actually, some search-n-replace seems to have been lost in translation :thinking: